### PR TITLE
Remove projectId & keyFilename requirements

### PR DESCRIPTION
**Pull request by eabadjiev**
Remove requirement for projectId & keyFilename, which when executed in Google managed environment can be inherited.

Also, a small but important change is to copy construct options before altering its values.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   },
   "devDependencies": {
     "codecov": "^1.0.1",
+    "expect": "^1.20.2",
```
Comment (flovilmart) [inline package.json]: not sure we need expect here

**Review by flovilmart — COMMENTED** [review]
Thanks for the PR! Can you address the comment please?

File: package.json (hunk 1 above)
Comment (eabadjiev) [reply package.json]: This dependency actually is not required, can be removed.

**Review by flovilmart — APPROVED** [review]

**Comment by flovilmart** [comment]
Thanks for that one!

**Comment by eabadjiev** [comment]
@flovilmart can you please release the changes in this pull request?

**Pull request merged**